app/app-entitlements-brave.plist

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
why do we have our own file that appears to be identical to app-entitlements-chrome.plist?

Reply:
@bridiver - `app-entitlements-chrome.plist` contains the `associated-domains.applinks.read-write` entitlement which we don't need. If we use the original file, it'll need more patches. https://source.chromium.org/chromium/chromium/src/+/master:chrome/app/app-entitlements-chrome.plist

Reply:
are we likely to miss out on future changes that we'll need?

Reply:
@bbondy - I was planning to add it to the CR rebase checklist here: https://github.com/brave/brave-browser/wiki/Chromium-rebases - Let me know if you have any concerns.

Reply:
We should be able to grant ourselves this entitlement and it would then be part of the provisioning profile. See https://github.com/brave/brave-browser/issues/8027#issuecomment-658867198 for more info. I am able to login as dev@brave.com but not sure how to add this ☹️ Might be something we have to email/ask about?

Reply:
@bsclifton - I'm happy to work on #8027 after resolving the hacker-one issues. Let me know if that works for you.

Reply:
@jumde SGTM! Feel free to assign yourself 😄